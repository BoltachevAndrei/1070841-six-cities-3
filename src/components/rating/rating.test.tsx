import * as React from 'react';
import * as renderer from 'react-test-renderer';
import Rating from './rating';

const rating = 3.5;

const type = `property`;

it(`Rating renders correctly`, () => {
  const tree = renderer
    .create(
        <Rating
          rating={rating}
          type={type}
        />
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
