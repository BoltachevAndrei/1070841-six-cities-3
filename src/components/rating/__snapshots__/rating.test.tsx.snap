// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Rating renders correctly 1`] = `
<div
  className="property__rating rating"
>
  <div
    className="property__stars rating__stars"
  >
    <span
      style={
        Object {
          "width": "70%",
        }
      }
    />
    <span
      className="visually-hidden"
    >
      3.5
    </span>
  </div>
  <span
    className="property__rating-value rating__value"
  >
    3.5
  </span>
</div>
`;
